.travis.yml

```diff
@@ -6,7 +6,32 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
-  - "3.6"
```

Comment:
I think that we need Python 3.6 here because it is needed for testing Django 1.11 and `master`. I think that the following diff contains the desired set f Python, Django, and matrix exclusions:

```diff
diff --git a/.travis.yml b/.travis.yml
index 01ff369..3db58d3 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -6,32 +6,28 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
 
 env:
   - DJANGO=1.8
   - DJANGO=1.9
   - DJANGO=1.10
   - DJANGO=1.11
+  - DJANGO=master
 
 matrix:
     fast_finish: true
-    include:
-      - python: "3.3"
-        env: DJANGO=1.8
-      - python: "3.5"
-        env: DJANGO=master
-      - python: "3.6"
-        env: DJANGO=1.11
-      - python: "3.6"
-        env: DJANGO=master
     exclude:
       - python: "2.7"
         env: DJANGO=master
       - python: "3.4"
         env: DJANGO=master
-      - python: "3.5"
-        env: DJANGO=master
-
+      - python: "3.6"
+        env: DJANGO=1.8
+      - python: "3.6"
+        env: DJANGO=1.9
+      - python: "3.6"
+        env: DJANGO=1.10
 
 install: pip install tox-travis
 
diff --git a/tox.ini b/tox.ini
index c0ee501..63019a7 100644
--- a/tox.ini
+++ b/tox.ini
@@ -20,6 +20,7 @@ basepython =
     py34: python3.4
     py35: python3.5
     py36: python3.6
+
 deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
```

This leads to the following set of builds, which line up with the Django FAQ (except Python 3.2 and 3.3, which aren't in the Travis CI configuration at all):

<img width="504" alt="screen shot 2017-03-28 at 20 31 27" src="https://cloud.githubusercontent.com/assets/43639/24433338/c814bcec-13f5-11e7-865a-5be21fd47a28.png">